src/render/decorations/CHyprGroupBarDecoration.cpp

```diff
@@ -151,19 +152,20 @@ void CHyprGroupBarDecoration::draw(CMonitor* pMonitor, float a) {
         CColor            color = m_dwGroupMembers[WINDOWINDEX].lock() == g_pCompositor->m_pLastWindow.lock() ? PCOLACTIVE->m_vColors[0] : PCOLINACTIVE->m_vColors[0];
         color.a *= a;
 
+        if (*PBOTTOMBAR)
+            g_pHyprOpenGL->renderRect(&rect, color);
+
         rect = {ASSIGNEDBOX.x + floor(xoff) - pMonitor->vecPosition.x + m_pWindow->m_vFloatingOffset.x,
                 ASSIGNEDBOX.y + ASSIGNEDBOX.h - floor(yoff) - ONEBARHEIGHT - pMonitor->vecPosition.y + m_pWindow->m_vFloatingOffset.y, m_fBarWidth,
                 (*PGRADIENTS || *PRENDERTITLES ? *PHEIGHT : 0)};
-        
-        g_pHyprOpenGL->renderRect(&rect, color, *PGROUPROUNDING); 
```

Comment:
Found out its cleaner to leave this where it originally was, the first `renderRect` is to render the thin bar, the rect is then recalculated as the main bar and gets rendered under the `if (*PGRADIENTS)` conditional, its better to keep them separated.